aws/src/integration/java/org/apache/iceberg/aws/glue/TestDynamoLockManager.java

```diff
@@ -153,9 +153,9 @@ public void testAcquireSingleProcess() throws Exception {
   @Test
   public void testAcquireMultiProcessAllSucceed() throws Exception {
     lockManager.initialize(ImmutableMap.of(
-        CatalogProperties.LOCK_ACQUIRE_INTERVAL_MS, "500",
-        CatalogProperties.LOCK_ACQUIRE_TIMEOUT_MS, "100000000",
-        CatalogProperties.LOCK_TABLE, lockTableName
+        LockManagerProperties.LOCK_ACQUIRE_INTERVAL_MS, "500",
+        LockManagerProperties.LOCK_ACQUIRE_TIMEOUT_MS, "100000000",
+        LockManagerProperties.LOCK_TABLE, lockTableName
```

Comment:
I don't think these properties need to be relocated. They are still properties that are passed to the catalog, so we should leave them in `CatalogProperties`.

Reply:
hmmm...not necessarily? users can create HadoopTables directly and then create HadoopTableOperations (assuming we agree on the current parameter passing mechanism that users pass in via Hadoop Configurations)

Reply:
These properties cannot change class for backwards compatibility. 

And I don't think it breaks definition of `CatalogProperties` to keep these configurations there, because there are 2 places using this, (1) hadoop catalog, (2) hadoop tables. For (1), it's still in a catalog, and for (2), you are technically not using these properties directly, but only using it with a pre-defined prefix. That is a custom definition in `HadoopTables` to use these properties in the specific way, I don't think we need to also move properties to another class to accommodate this use case.